add-ons/fabric8-launcher/fabric8-launcher.addon

```diff
@@ -17,4 +17,4 @@ oc new-project #{LAUNCHER_PROJ}
 oc create -n #{LAUNCHER_PROJ} -f https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master/openshift/launcher-template.yaml
 
 # instantiate launcher
-oc new-app fabric8-launcher --name=launcher -n #{LAUNCHER_PROJ} -p LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=#{GITHUB_USERNAME} -p LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=#{GITHUB_TOKEN} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_USERNAME=#{OPENSHIFT_USERNAME} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD=#{OPENSHIFT_PASSWORD}  
+oc new-app fabric8-launcher --name=launcher -n #{LAUNCHER_PROJ} -p LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=#{GITHUB_USERNAME} -p LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=#{GITHUB_TOKEN} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_USERNAME=#{OPENSHIFT_USERNAME} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD=#{OPENSHIFT_PASSWORD} -p LAUNCHER_KEYCLOAK_URL= -p LAUNCHER_KEYCLOAK_REALM=
```

Comment:
Here `LAUNCHER_KEYCLOAK_URL=` and `LAUNCHER_KEYCLOAK_REALM=` value is not defined, does it as expected if yes then why adding it here?

Reply:
The template has default values for these parameters that need to be blank if you are not using KC, I believe that's the reason why

Reply:
@gastaldi Then why not remove it from here or if you provide user to overwrite this with own values then better to put in `Required-Vars` section and have those same default value in`Var-Defaults` section ?

Reply:
They are not really required, only in the case of installing it without Keycloak support. +1 to move that to `Var-Defaults`

Reply:
@praveenkumar is there a way to pass an empty value via Var-Defaults?

I have tried the following, but all of them are taken as literal values or are considered malformed:
* KEYCLOAK_URL="",KEYCLOAK_REALM=""
* KEYCLOAK_URL='',KEYCLOAK_REALM=''
* KEYCLOAK_URL=,KEYCLOAK_REALM=
* KEYCLOAK_URL=null,KEYCLOAK_REALM=null
* KEYCLOAK_URL=NULL,KEYCLOAK_REALM=NULL
* KEYCLOAK_URL=nil,KEYCLOAK_REALM=nil

Reply:
@rhoads-zach I still don't get why you want to have an empty value in the Var-Defaults, If you don't need any value then what is the point of having those variables?  From @gastaldi comment, I understood that these are only required when a user wants to have Keycloak support so doesn't it set something in the background when you don't pass it as variable or no default value for these in the template?

Reply:
@praveenkumar last i checked, if the user does not specify the `LAUNCHER_KEYCLOAK*` values when running `oc new-app fabric8-launcher...`, it defaults to what the developers.redhat.com/launch uses. We want the values to be empty when installing on minishift. @gastaldi am i correct?

Reply:
@rhoads-zach that's correct.

@praveenkumar is there any way to specify an empty value for the variable using `Var-Defaults`?

Reply:
@gastaldi At this moment we don't have any capability to provide an empty value for `Var-Defaults` because we thought this is something never required, I will create an issue on the minishift side for this feature, meanwhile I think you are better off with what you have atm.

Reply:
@praveenkumar thank you for the feedback.

If you are fine with the current state of the PR, please merge.

Also, when you add the empty param feature, please tag me on the issue/PR and i'll do a follow up PR to use it.